docs/development_suite/identity-and-access-management/user-management.md

```diff
@@ -107,21 +105,55 @@ The Company Owner can add a new User to a Company by pressing the *Add user* but
   </div>
 </div>
 
-The provided email will be searched throughout existing Users and, if a match is found, the User will be invited to the Company with the defined Roles.
+The provided email will be searched throughout existing users and, if a match is found, the user will be invited to the Company with the defined roles.
 
 :::info
-If the User does not exist in the Platform, please open a Service Request.
+If the user does not exist in the Platform, please open a Service Request.
 :::
 
 
 ### Editing a User Role at Project level
 
-A User Role in the Project or any of the Project Runtime Environments can be modified. To do so, just open the editing dialog and select the proper Role for the Project itself or for each Runtime Environment.
+A user role in the Project or any of the Project Runtime Environments can be modified. To do so, just open the editing dialog and select the proper role for the Project itself or for each Runtime Environment.
 
 <div style={{display: 'flex', justifyContent: 'center'}}>
   <div style={{display: 'flex', width: '600px'}}>
 
-![Edit Project User](./img/user-management/edit_project_user.png)
+![Edit Project User](./img/user-management/edit_at_project_level.png)
 
   </div>
-</div>
\ No newline at end of file
+</div>
+
+## How to best manage your users
+
+Are you wondering whether organizing your users through individual role assignments is the smartest and most effective way to manage the user access to your Company? Or whether the best solution would be to organize them into groups instead?
+There is no right or wrong answer to this question: it depends on your organizational needs.  
+Here you can find some helpful hints that can help you find an answer that better suits your own requirements.
+
+For example, if you need to provide the same access privileges to multiple users in your Company, for sure the simplest and most efficient way is to manage them through groups.  
+
+Imagine that you have to manage a team composed by a large number of developers. You can create a group (e.g., "My Company Developers"), add them all as group members, and assign the most appropriate role (e.g., `Developer` role) to the group. This will result in all your users becoming users of the Company with Developer role by virtue of being part of that group of developers.
```

Comment:
```suggestion
On this purpose, imagine that you have to manage a team composed by a large number of developers. You can create a group (e.g., "My Company Developers"), add them all as group members, and assign the most appropriate role (e.g., `Developer` role) to the group. This will result in all your users becoming users of the Company with Developer role by virtue of being part of that group of developers.
```